src/validation/validators/matchField/MatchFieldValidation.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { MatchFieldError } from "@/validation/errors/MatchFieldError";
+import { MatchFieldValidation } from "./MatchFieldValidation";
+import faker from "faker";
+
+const ONLY_DIGITS_REGEX = /^[0-9]*$/;
+const CAPITAL_LETTERS_REGEX = /[A-Z]+/;
+
+const makeDigitsSut = (fieldName: string): MatchFieldValidation =>
+  new MatchFieldValidation(fieldName, ONLY_DIGITS_REGEX);
+
+const makeCapitalLettersSut = (
+  fieldName: string,
+  ignoreCase: boolean,
+  message?: string
+): MatchFieldValidation =>
+  new MatchFieldValidation(
+    fieldName,
+    CAPITAL_LETTERS_REGEX,
+    ignoreCase,
+    message
+  );
+
+describe("MatchFieldValidation", () => {
+  test("Should return error if valid is invalid", () => {
+    const field = faker.database.column();
+    const sut = makeDigitsSut(field);
+    const error = sut.validate({ [field]: faker.random.alphaNumeric(4) });
+
+    expect(error).toEqual(new MatchFieldError());
+  });
+
+  test("Should return falsy if value is valid", () => {
```

Comment:
I think it should be 'Should return falsy if match is valid for Invalid Digit Regex'